pages/1.14/administering-clusters/draining-a-node/index.md

```diff
@@ -0,0 +1,77 @@
+    ```
+
+Overall progress of draining on the node may be monitored with the following command:
+
+    ```bash
+    dcos node list
+    ```
+
+Locate the relevant agent node in the list and see its `STATUS` field. The usual status of a node will be `ACTIVE`. When draining is initiated on the node, it will transition to `DRAINING`. Once all tasks have been killed and their terminal status updates have been acknowledged by the relevant services, and once all resource operations on the node have finished, it will transition to `DRAINED`. When the node is in the `DRAINED` state, draining is complete.
+
+If the `--decommission` option was included in the initial drain command, the node's metadata will be automatically removed from the cluster when draining is complete. Otherwise, the cluster will wait for the node to be manually reactivated when any planned maintenance or other update procedures are finished. Before performing maintenance, stop the `dcos-mesos-agent` systemd unit on the node so that the agent no longer attempts to register with the master. When maintenance is complete and you are ready to reactivate the node and allow it to rejoin the cluster, you may do so with the following command:
+
+    ```bash
+    dcos node reactivate <mesos-id>
+    ```
+
+Then start the `dcos-mesos-agent` systemd unit on the node and the agent will rejoin the cluster.
+
+## Monitoring Draining with the UI
+
+You may also monitor the draining of an agent via the DC/OS UI. To do so, navigate to the `Nodes` overview pane. The `Status` column will list either `Active`, `Draining`, or `Drained`, similar to the output of the CLI's `node list` command.
+
+## Notes Regarding Node Draining
+
+Once draining is initiated on an agent node, it cannot be cancelled. This command should be used with caution, especially if executed with the `--decommission` option, as this results in the node's permanent removal from the cluster when draining is complete.
+
+If a node becomes stuck in the `DRAINING` state and does not transition to `DRAINED` after the expected duration (either the duration specified in the `--timeout` argument, or the longest kill grace period of the tasks running on the node), then it should be inspected to determine the cause. If all tasks on the node have terminated, then it's possible that either a terminal task status update has not been acknowledged by the relevant service, or a resource operation on that node has not finished. To rule out the former, inspect the relevant services which were running tasks on the node and ensure that they are running and available. To rule out the latter, the `GET_OPERATIONS` call may be used on the agent node in order to inspect any currently-pending resource operations. If a resource operation is still in the `Pending` state, the operation may be long-running and take some time to complete, or it may be stuck due to some issue with the storage backend which is actually performing the operation.
```

Comment:
You should probably link to Mesos documentation about how one can issue a `GET_OPERATIONS` call.